test/ci/kokoro/run_integ_tests.sh

```diff
@@ -54,18 +54,16 @@ function latest_python_release {
 function install_pyenv {
   # Install pyenv if missing.
   if ! [ "$(pyenv --version)" ]; then
-    # For now, only doing this on mac,
-    # beacuse that was the only place where it appeared to be missing.
-    if [[ $KOKORO_JOB_NAME =~ "macos" ]]; then
-      brew update
-      brew install pyenv
-      eval "$(pyenv init -)"
-    fi
+    # MacOS VM does not have pyenv installed by default.
+    git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+    export PYENV_ROOT="$HOME/.pyenv"
+    export PATH="$PYENV_ROOT/bin:$PATH"
+    eval "$(pyenv init --path)"
```

Comment:
Why not just run `pyenv init --path`?

Reply:
The way that command works is that it will output a bunch of other commands, and we want to execute those.